swri_math_util/include/swri_math_util/ransac.h

```diff
@@ -49,14 +49,14 @@ namespace swri_math_util
     explicit Ransac(RandomGeneratorPtr rng = RandomGeneratorPtr()) : rng_(rng) {}
 
     ModelType FitModel(
-      const DataType& data,
+      Model& model,
       double max_error,
       double confidence,
+      int32_t min_iterations,
```

Comment:
min_iterations is a useful addition, but it seems like there should be a default (or hint on how this number should be chosen).  The prior version effectively functioned as min_iterations = 0, right?